### PR TITLE
Newsletter for Feb 2021 + RSES info/showcase event

### PR DESCRIPTION
**Pull request by willfurnass**

### CODE REVIEW

File: _newsletters/2021-02-08-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+To receive this newsletter as an email each month, please sign up to our [Google Group][rses-mail-list].
+
+*All times are GMT (UTC+00)*
+
+
+### News
+
+An [**International Council of RSE Associations**](https://researchsoftware.org/2021/01/27/introducing-the-international-council-of-RSE-associations.html) has been founded
+to help the various existing national Research Software Engineering associations work better together and
```
Comment (Robadob) [inline _newsletters/2021-02-08-newsletter.md]: Is there a reason RSE hasn't been abbreviated here, but it has in the prior and following lines it has? Initial read of the block felt a bit too wordy and this would probably help. Alternative being refer to `national counterparts` or similar instead.

File: _newsletters/2021-02-08-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+Our local chapter of the [UK Reproducibility Network](https://www.ukrn.org/) (UKRN)
+is being rebranded the [**Open Research Working Group**](https://twitter.com/OpenResShef/status/1351141482808414210) and will be run by new leads
+([Claudia von Bastian ](https://www.sheffield.ac.uk/psychology/people/academic/claudia-von-bastian) and [Jim Utley](https://www.sheffield.ac.uk/architecture/people/academic-staff/jim-uttley)).
+Through this group members will share good practice, events and facilitating conversation about open research.
+
+---
+
+
+**Stanford have conducted a survey on the use and creation of research software**; see the [results here](https://stanford-rc.github.io/stanford-software-survey/).
+The RSE team at Sheffield conducted a similar survey here recently and will be sharing the results soon.
```
Comment (Robadob) [inline _newsletters/2021-02-08-newsletter.md]: 'here' has the potential to be vague.

File: _newsletters/2021-02-08-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+---
+
+
+**Stanford have conducted a survey on the use and creation of research software**; see the [results here](https://stanford-rc.github.io/stanford-software-survey/).
+The RSE team at Sheffield conducted a similar survey here recently and will be sharing the results soon. 
+
+---
+
+The Research IT team in IT Services are now on Twitter ([@RIT_Sheffield](https://twitter.com/RIT_Sheffield)).
```
Comment (Robadob) [inline _newsletters/2021-02-08-newsletter.md]: Worth briefly clarifying the distinction between RIT and RSE? Might not be obvious to all.

File: _newsletters/2021-02-08-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+
+The Sheffield RSE team is running two full-day practial workshops on **Deep Learning, focussing on the use of TensorFlow with Python and R (plus Keras)**. Content includes:
+
+  * Introduction to Regression and Classification using neural networks
+  * Image classification with Convolutional neural networks
+  * Speeding up the training process of your own models using existing neural network
+
+Course Registration:
+
+  * [Deep Learning with Tensorflow in Python (2021-02-11)](https://www.eventbrite.co.uk/e/131825719275)
```
Comment (Robadob) [inline _newsletters/2021-02-08-newsletter.md]: This has sold out.

Comment (Robadob) [reply _newsletters/2021-02-08-newsletter.md]: (e.g. add sold out in italics, and that we hope to run again in future as it has been very popular?)

File: _newsletters/2021-02-08-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+
+---
+
+The [Software Sustainability Institute](https://www.software.ac.uk/) will 
+host its first online [**Research Software Camp on research accessibility**](https://www.software.ac.uk/RSCamp-research-accessibility) 
+from Monday 22 February to Friday 5 March 2021. 
+This is a free online event encompassing live sessions, web resources and social media activities. 
+It will focus on different aspects of research accessibility, ranging from making tools, techniques, datasets, software and protocols used in research reproducible, open and sustainable, 
+to exploring formats and tools to facilitate and improve accessibility for people with disabilities. 
+They welcome anyone involved with research and research software from all career stages to attend any or all of the live sessions, and take part in our social media and website activities.
```
Comment (Robadob) [inline _newsletters/2021-02-08-newsletter.md]: **our** social media and website activities.

File: _newsletters/2021-02-08-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+at which you could get support from experienced, knowledgable mentors. 
+This is the successor to the GPU hackathon held in Sheffield in recent years.
+See these [blog](https://rse.shef.ac.uk/blog/2019-10-10-gpu-hackathon/) [posts](https://rse.shef.ac.uk/blog/2019-12-16-gpu-hackathon/)
+for what a GPU hackathon can offer.
+Application deadline: 2021-02-19.
+Event itself: 2021-04-19, 2021-04-26, 2021-04-27, 2021-04-28.
+
+---
+
+External workshop on **Sustainable Software Development for Researchers**: At this workshop, attendees will learn about the basic steps to prepare code for sharing with others and make it ready for citation in a research paper.
```
Comment (Robadob) [inline _newsletters/2021-02-08-newsletter.md]: reads weird, "learn about the basic steps to prepare code for sharing with others and make it ready for citation in a research paper".
prepare means the same as "make it ready" so the latter is redundant, removing make it ready might be better?

File: _newsletters/2021-02-08-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+
+This workshop is run by [Carina Haupt](https://www.dlr.de/sc/en/desktopdefault.aspx/tabid-1192/1635_read-28643/sortby-lastname/), Head of Group Software at the German Aerospace Centre. It consists of two sessions taking place on Thursday 25 February 13:00 to 17:00 GMT and Friday 26 February 9:00 to 12:30 GMT -- participants attend both sessions. [Register on Eventbrite](https://www.eventbrite.co.uk/e/research-software-camp-sustainable-software-development-for-researchers-tickets-139626658093?aff=erelpanelorg).
+
+---
+
+External workshop on **Boosting your research reproducibility with Binder**: "During this free workshop, we will discuss reproducible computing environments and why they're important, show examples of others' projects in [mybinder.org](https://mybinder.org/) and help you learn how to prepare a binder-ready project." The workshop is run by The Turing Way, and will take place on Wednesday 3 March 2021, 10:00 to 16:30 GMT. [Register on Eventbrite.](https://www.eventbrite.co.uk/e/research-software-camp-boost-your-research-reproducibility-with-binder-tickets-138852787425)
```
Comment (Robadob) [inline _newsletters/2021-02-08-newsletter.md]: **we** will discuss

**Review by Robadob — COMMENTED** [review]
Seems through in general.

I get the feeling that your summaries for events are far more detailed than usual. Possibly because you've copied/reduced their own blurb. Nothing wrong with that, but in a few places it talks about "we" or "our" that is a 3rd party even unrelated to our team.

**Comment by willfurnass** [comment]
Thanks for the review @Robadob!

**Comment by willfurnass** [comment]
Lazily bundled a new Event into this PR for the RSE Sheffield info/showcase event in March.

**Comment by willfurnass** [comment]
Need to finalise info on next LB event before publishing: may want/need to postpone next LB to March or change theme.
